src/reader/widgets/ReaderWidget.vue

```diff
@@ -98,6 +165,21 @@
           ? new URN(this.$route.query.urn)
           : this.$store.getters[`${MODULE_NS}/firstPassageUrn`];
       },
+      coordinatesList() {
+        return this.gqlData.passageTextParts.edges
+          .map(e => {
+            return e.node.tokens.edges.map(te => {
+              return te.node.namedEntities.edges
+                .map(ne => ne.node)
+                .filter(n => n.kind === 'PLACE' && n.data.coordinates)
+                .map(n => {
+                  return n.data.coordinates.split(', ').map(c => parseFloat(c));
+                });
+            });
+          })
+          .flat()
```

Comment:
is this a duplicated call to `flat`?

Reply:
you have to call `flat()` for each level of _flatness_